google-cloud-dlp/lib/google/privacy/dlp/v2/dlp_pb.rb

```diff
@@ -5,6 +5,7 @@
 require 'google/protobuf'
 
 require 'google/api/annotations_pb'
+require 'google/api/resource_pb'
```

Comment:
This annotation proto file does not exist in the current common protos gem. We need a new common protos gem to merge and release this.